packages/renderer/src/lib/ui/DetailsPage.svelte

```diff
@@ -21,46 +21,49 @@ function handleKeydown(e: KeyboardEvent) {
 
 <svelte:window on:keydown="{handleKeydown}" />
 
-<div class="w-full h-full">
-  <div class="flex h-full flex-col">
-    <div class="flex w-full flex-row">
-      <div class="w-full pl-5 pt-4">
-        <div class="flex flew-row items-center text-sm text-gray-700">
-          <Link aria-label="back" internalRef="{$lastPage.path}" title="Go back to {$lastPage.name}"
-            >{$lastPage.name}</Link>
-          <div class="mx-2">&gt;</div>
-          <div class="font-extralight" aria-label="name">{$currentPage.name}</div>
+<div class="flex flex-col w-full h-full shadow-pageheader">
+  <div class="flex flex-row w-full h-fit px-5 py-4">
+    <div class="flex flex-col w-full h-fit">
+      <div class="flex flew-row items-center">
+        <Link class="text-sm" aria-label="back" internalRef="{$lastPage.path}" title="Go back to {$lastPage.name}"
+          >{$lastPage.name}</Link>
+        <div class="text-md mx-2 text-gray-700">></div>
```

Comment:
any reason why `&gt;` is replaced with `>` ?

Reply:
Purely due to using multiple workspaces and accidentally undoing my own changes... sorry, fixed now.